src/model/liverange.js

```diff
@@ -40,6 +40,13 @@ export default class LiveRange extends Range {
 		this.stopListening();
 	}
 
+	/**
+	 * @inheritDoc
```

Comment:
You send people to `Range#is()` which documents that:

> Checks whether given object is of `range` type.

Which means that you still don't know how to use this method to check for a live range